src/sync/backfill-since-date-calc.ts

```diff
@@ -0,0 +1,48 @@
+import { SyncType } from "~/src/sync/sync.types";
+
+export const calcNewBackfillSinceDate = (
+	existingBackfillSince: Date | undefined,
+	commitsFromDate: Date | undefined,
+	syncType: SyncType | undefined,
+	isInitialNewSync: boolean | undefined
+): Date | undefined  => {
```

Comment:
Do we really need `syncType` here? Can't we decide on the basis of `isInitialNewSync`?

Reply:
If partial, we should ignore it for safe side. So yes, need synctype.

Reply:
keep synctype!! i want to start using Partial a lot so will be more relevant in time

Reply:
LOL, 👍 
Close this convo